js/base-weather.js

```diff
@@ -32,6 +32,7 @@ function baseWeather () {
 			"Waves": new Image(),
 			"Ripples": new Image(),
 			"Blood Rain": new Image(),
+			"Custom": new Image(),
 		};
 		IMAGES.Rain.src = "https://i.imgur.com/lZrqiVk.png";
```

Comment:
come to think of it, do we have these in the repo anywhere? If not, we're at risk of them disappearing someday
Might be best to commit them to e.g. `img/weather/*` and reference the `raw.github. ...` URLs here (i.e. `https://raw.githubusercontent.com/TheGiddyLimit/betterR20/development/img/weather/rain.png`)